src/app/map/components/drawing-symbols/drawing-symbols-collection/drawing-symbols-collection.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="drawing-symbols-collection__grid">
+  @for (drawingSymbol of items; track $index) {
+    <label [attr.for]="drawingSymbol.name" class="drawing-symbols-collection__grid__item">
+      <input
+        type="radio"
+        class="sr-only"
+        [name]="groupName"
+        [id]="drawingSymbol.name"
+        [checked]="isSelected(drawingSymbol.item)"
+        (change)="onValueChange(drawingSymbol.item)"
+      />
+      <img
+        loading="lazy"
```

Comment:
I think you already mentioned that this does not work in some browsers. When the component is opened in Chrome, it shoots a gazillion requests to Esri and loads all the icons at once. This might have an impact to the perceived lag as well.

I mentioned in another file that this is also due to the eager loading of the `ExpandableListItem` component; so this should definitely be fixed.